xgcm/grid.py

```diff
@@ -1399,7 +1401,12 @@ def interp_like(self, array, like):
             except KeyError:
                 continue
             if position_like != position_array:
-                array = self.interp(array, axname)
+                array = self.interp(
+                    array,
+                    axname,
+                    fill_value=fill_value.get(axname, None),
```

Comment:
```suggestion
                    fill_value=fill_value.get(axname),
```

`None` is the default value returned